Dockerfile.windows

```diff
@@ -12,9 +12,9 @@ RUN apt update && \
 RUN update-alternatives --set x86_64-w64-mingw32-g++ $(which x86_64-w64-mingw32-g++-posix) && \
     update-alternatives --set x86_64-w64-mingw32-gcc $(which x86_64-w64-mingw32-gcc-posix)
 
-RUN git clone -b v0.18.2.0 --depth 1 https://github.com/monero-project/monero && \
+RUN git clone -b v0.18.4.0 --depth 1 https://github.com/monero-project/monero && \
```

Comment:
Did the Windows build ship with `v0.18.2.0` all this time? Wow.

Reply:
The dependencies for monero-wallet-gui were on v0.18.2.0 depends, yes. The daemon was taken from our CLI release builds.